install.py

```diff
@@ -13,8 +13,9 @@ def log(string):
     print(string, flush=True)
 
 
-def retryUntilSuccess(func):
-    while True:
+def retryUntilSuccess(func, timeout = 0):
+    end = time.time() + timeout
+    while timeout <= 0 or time.time() < end:
```

Comment:
Apologies for the bug here. Something funky happened with the registry in my VM and this initially got through, but it should be fine now.